test/EndToEndTests/Tests/Client/Microsoft.OData.Client.E2E.Tests/Common/Server/Default/DefaultDataModel.cs

```diff
@@ -73,6 +74,7 @@ public class Person
         public Address? HomeAddress { get; set; }
         public GeographyPoint? Home { get; set; }
         public Person? Parent { get; set; }
+        public DateTime UpdatedTime { get; set; }

```

Comment:
why do you use 'DateTime'? It's not compatible to OData spec. Maybe to use "DateTimeOffset"?